clients/typescript/src/util/tablename.ts

```diff
@@ -14,7 +14,26 @@ export class QualifiedTablename {
   }
 
   toString(): string {
-    return `${this.namespace}.${this.tablename}`
+    // Escapes double quotes because names can contain double quotes
+    // e.g. CREATE TABLE "f""oo" (...) creates a table named f"oo
+    return `"${escDoubleQ(this.namespace)}"."${escDoubleQ(this.tablename)}"`
+  }
+
+  static parse(fullyQualifiedName: string): QualifiedTablename {
+    try {
+      const [_, namespace, tablename] = /"(.*)"\."(.*)"/.exec(
```

Comment:
Counterexample: `"main"".""".""".""body"` which is a valid SQL value with (in unescaped form) schema `main"."` and table name `"."body`, but won't be parsed as such. Edge case for sure, but also makes invalid strings valid, like `"main""."s".""".""body"`. We also should match string boundaries.

So, to never return to this again, allow only paired double quotes within the quotes. Also identifers can't be empty: 

```suggestion
      const [_, namespace, tablename] = /^"((?:[^"]|"")+)"\."((?:[^"]|"")+)"$/.exec(
```